Content/Data/Items-External.xml

```diff
@@ -529,4 +529,43 @@
     <source>(0,0,45,45)</source>
     <offset>(-10,-10)</offset>
   </item>
+<<<<<<< HEAD
```

Comment:
Merge line endings are here. Need to clean those up and make sure the xml is valid.